src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +52,68 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={value}
+              onChange={(event) => setValue(event.target.value)}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {showError && isRequested && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button is-light',
+                { 'is-loading': isNowRequested },
+              )}
+              onClick={(event) => {
```

Comment:
Consider using 'onSubmit' on the form element instead of using 'onClick' on the submit button.